Add abbreviations from UD_Swedish-Talbanken

EmilStenstrom opened the pull request:
## Description
This is part of #2608. It reduces the tokenization errors from 216 -> 196. The remaining errors has to do with token splitting, I'll handle that in a separate PR. All added abbreviations are from UD_Swedish-Talbanken.

### Types of change
Bug fix.

## Checklist
<!--- Before you submit the PR, go over this checklist and make sure you can
tick off all the boxes. [] -> [x] -->
- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

honnibal commented:
Thanks! Fixed a small merge conflict with the previous PR. Just waiting on the CI checks to pass.

EmilStenstrom commented:
@honnibal I think you accidentally removed my changes, making the build fail. I've rebased my branch on master, and fixed the merge conflict. Just waiting for the CI to complete again now! :)
